lib/file_manipulation.rb

```diff
@@ -0,0 +1,22 @@
+class FileManipulation
+  
+  def initialize(path_to_file)
+    @path_to_file = path_to_file
+    @file = File.open(path_to_file)
+    
+    begin
+      file = open(@path_to_file)
+      file.close
+    rescue => exception
+      puts "#{$!.class}: #{$!.message}"
+    end
+  end
+
+  def read_especific_line(line_number)
+    IO.readlines(@path_to_file)[line_number]
+  end
+
+  def exists
```

Comment:
por padrão no ruby sempre que um função devolve um boolean, essa função tem o interrogação
```suggestion
  def exists?
```